Recurly/Resources/Error.cs

```diff
@@ -21,7 +21,7 @@ public class Error : Resource
 
         /// <value>Parameter specific errors</value>
         [JsonProperty("params")]
-        public List<Dictionary<string, string>> Params { get; set; }
```

Comment:
Perhaps I can put in a hack to preserve this one so to limit the scope of the breaking change.